crates/primitives/src/specification.rs

```diff
@@ -64,6 +64,7 @@ pub enum SpecId {
     CANCUN = 20,
     ECOTONE = 21,
     PRAGUE = 22,
+    FJORD = 23,
```

Comment:
Should FJORD be after or before PRAGUE? In secp256r1 precompile PR it was before.

Reply:
Put FJORD before PRAGUE as Prague is still not activated